code/src/layouts/header/components/header-toolbar/HeaderToolbar.tsx

```diff
@@ -0,0 +1,52 @@
+import ShoppingCartIcon from "@mui/icons-material/ShoppingCart";
+
+import AppBadge from "@/components/app-badge/AppBadge";
+import AppBox from "@/components/app-box/AppBox";
+import AppButton from "@/components/app-button/AppButton";
+import AppContainer from "@/components/app-container/AppContainer";
+import AppIconButton from "@/components/app-icon-button/AppIconButton";
+import AppLogo from "@/components/app-logo/AppLogo";
+import AppLink from "@/components/app-link/AppLink";
+import AppTypography from "@/components/app-typography/AppTypography";
```

Comment:
I suggest to create `index` file for re-export files from `@/components` for example

instead of these imports we'll only have one with imports files like
`import { AppBadge, AppBox ...} from "@/components"`
imports just from one place
it'll shorten the code

Reply:
I think we should follow one convention and not jumping from one approach to another. On s2s we did not use index file

Reply:
> I think we should follow one convention and not jumping from one approach to another. On s2s we did not use index file. I would be interesting for us if you can implement it

Sure. It's a good approach to imports all files from one place and will shorten the code.
I'll refactore it